wagtail/core/blocks/field_block.py

```diff
@@ -594,18 +596,23 @@ class Meta:
 
 
 class PageChooserBlock(ChooserBlock):
-
-    # TODO: rename target_model to page_type
-    def __init__(self, target_model=None, can_choose_root=False,
-                 **kwargs):
+    def __init__(self, page_type=None, can_choose_root=False, target_model=None, **kwargs):
         if target_model:
+            warnings.warn(
+                "PageChooserBlock's `target_model` parameter is deprecated. "
+                "Please use `page_type` instead.",
```

Comment:
I could not find an example of deprecation warning for renaming a method's parameter so I don't know whether this is consistent with old warnings.